src/server/scripts/Commands/cs_deserter.cpp

```diff
@@ -89,26 +89,15 @@ class deserter_commandscript : public CommandScript
     * .deserter bg add 3600 (one hour)
     * @endcode
     */
-    static bool HandleDeserterAdd(ChatHandler* handler, char const* args, bool isInstance)
+    static bool HandleDeserterAdd(ChatHandler* handler, uint32 time, bool isInstance)
```

Comment:
Might be useful to allow taking std::chrono types to make the unit of time explicit?

Reply:
Yes, that would be definitely nice. I would like do that in a second step since we have to implement duration string parsing and the consumer APIs have to be converted as well.

Reply:
sure, works for me